usr/src/uts/armv8/os/ddi_impl.c

```diff
@@ -1442,7 +1442,7 @@ get_boot_properties(void)
 				(void) e_ddi_prop_update_int64_array(
 				    DDI_DEV_T_NONE, devi, property_name,
 				    bop_staging_area,
-				    length / sizeof (int64_t));
+				    BYTES_TO_1275_CELLS(length));
```

Comment:
This isn't equivalent given the `int64_t`, and I'm not sure the previous two macro uses are actually correct, even though they produce the right result.